config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -11,155 +11,160 @@ spec:
+                handled by an external CSI Volume Driver (Alpha feature).
+              properties:
+                creationTime:
+                  description: Timestamp when the point-in-time snapshot is taken
+                    by the underlying storage system. This timestamp will be generated
+                    and returned by the CSI volume driver after the snapshot is cut.
+                    The format of this field is a Unix nanoseconds time encoded as
+                    an int64. On Unix, the command `date +%s%N` returns the current
+                    time in nanoseconds (aka, epoch time) since 1970-01-01 00:00:00
+                    UTC. This field is required in the CSI spec however made optional
+                    here to support static binding.
+                  format: int64
+                  type: integer
+                driver:
+                  description: Driver is the name of the driver used to create a physical
+                    snapshot on underlying storage system. This MUST be the same name
+                    returned by the CSI GetPluginName() call for that driver. Required.
+                  type: string
+                restoreSize:
+                  description: When restoring a volume from a snapshot, the volume
+                    size needs to be equal to or larger than the RestoreSize if it
+                    is specified. If RestoreSize is set to nil, in the dynamic snapshot
+                    creation case, it means that the underlying storage system does
+                    not have this information available; in the static binding case,
+                    this piece of information is not available.
+                  format: int64
+                  type: integer
+                snapshotHandle:
+                  description: SnapshotHandle is the unique id returned from the underlying
+                    storage system by the CSI driver's CreationSnapshot gRPC call.
+                    It serves as the only and sufficient handle when communicating
+                    with underlying storage systems via CSI driver for all subsequent
+                    calls on the specific VolumeSnapshot Required.
+                  type: string
+              required:
+              - driver
+              - snapshotHandle
+              type: object
+            deletionPolicy:
+              description: DeletionPolicy defines whether a VolumeSnapshotContent
+                and its associated physical snapshot on underlying storage system
+                should be deleted or not when released from its corresponding VolumeSnapshot.
+                If not specified, the default will be VolumeSnapshotContentRetain
+                for static binding, and VolumeSnapshotContentDelete for dynamic snapshot
+                creation.
+              type: string
+            persistentVolumeRef:
+              description: PersistentVolumeRef represents the PersistentVolume that
```

Comment:
I'm still unclear about why we need this.  What happens if the PV is deleted and doesn't exist anymore?

Reply:
this will be removed. I will fill you in next week what have been discussed today.